lib/appsignal/extension.rb

```diff
@@ -28,5 +28,11 @@ def method_missing(m, *args, &block)
         # Do nothing if the extension methods are not loaded
       end
     end
+
+    class Data
+      def inspect
+        "#<#{self.class.name}:#{self.object_id} #{self.to_s}>"
```

Comment:
no need for `self.` with `object_id` and `to_s` methods I think. It is needed for `self.class`